packages/formation-react/src/components/HelpMenu/HelpMenu.jsx

```diff
@@ -21,12 +22,15 @@ class HelpMenu extends React.Component {
             <b>Call the VA.gov Help Desk</b>
           </p>
           <p>
-            <a href="tel:18555747286">1-855-574-7286</a>
+            <Telephone contact={CONTACTS.HELP_DESK} />
           </p>
           <p>
-            TTY: <a href="tel:+18008778339">1-800-877-8339</a>
+            TTY:{' '}
+            <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
+          </p>
+          <p>
+            We’re here <abbr title="24 hours a day, 7 days a week">24/7</abbr>
```

Comment:
```suggestion
            We’re here <abbr title="24 hours a day, 7 days a week">24/7</abbr>.
```

:shrug:

Reply:
I added the `<abbr>` wrap as it was recommended by Jennifer and is already in use in our HLR form - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/996/content/GetFormHelp.jsx#L12